src/artemis/devices/det_dist_to_beam_XY_converter.txt

```diff
@@ -1,10 +1,10 @@
 #Table giving position of beam X and Y as a function of detector distance
 #Units mm mm mm
 # Eiger values
-# distance beamY beamX (values from mosflm)
+# distance beamX beamY (values from mosflm)
```

Comment:
**should**: Can you make the same change in GDA?

Reply:
I've made ticket MXGDA-3879 to address this.